internal/pkg/manifest/http.go

```diff
@@ -88,38 +88,65 @@ func ipNetP(s string) *IPNet {
 	return &ip
 }
 
+// AdvancedAlias represents advanced alias configuration.
+type AdvancedAlias struct {
+	Alias      *string `yaml:"value"`
```

Comment:
Can we use `name` to match the AWS API 🥺 ? https://aws.amazon.com/premiumsupport/knowledge-center/alias-resource-record-set-route53-cli/
```suggestion
	Alias      *string `yaml:"name"`
```